lib/matplotlib/widgets.py

```diff
@@ -1997,7 +1997,7 @@ def __init__(self, ax, onselect, drawtype='box',
         and *minspanx* will be interpreted in the same coordinates as
         the x and y axis. If 'pixels', they are in pixels.
 
-        *button* is a list of integers indicating which mouse buttons should
+        *button* is the list of (integer) mouse buttons which should
```

Comment:
```suggestion
        *button* is the list of `.MouseButton`\s which should
```
Will additionally need to make it a raw string for the escape backslash.